tests/functional/spec/final-confirmation.spec.js

```diff
@@ -0,0 +1,46 @@
+import chai from 'chai'
+import {startQuestionnaire} from '../helpers'
+import MandatoryRadioPage from '../pages/surveys/radio/mandatory-radio.page'
+import OptionalRadioPage from '../pages/surveys/radio/optional-radio.page'
+import Confirmation from '../pages/confirmation.page'
+import ThankYou from '../pages/thank-you.page'
+
+const expect = chai.expect
+
+describe('Final confirmation before submit', function () {
+
+    var confirmation_schema = 'test_final_confirmation.json';
+
+    it('Given I successfully complete a questionnaire, when I submit the page, then I should be prompted for confirmation to submit.', function () {
+        //Given
+        startQuestionnaire(confirmation_schema)
+
+        // When
+        MandatoryRadioPage.clickOther().setOtherInputField('Hello').submit()
+
+        //Then
+        expect(Confirmation.getSubtitle()).to.contain('Thank you for your answers, do you wish to submit')
+
+        Confirmation.submit()
+
+        expect(ThankYou.getTitle()).to.contain('Submission Successful')
+    })
+
+
+    it('Given I successfully complete a questionnaire and an on the confirmation page, when I click change answers, then I should be returned to the questionnaire', function () {
+        //Given
+        startQuestionnaire(confirmation_schema)
+
+        // When
+        MandatoryRadioPage.clickOther().setOtherInputField('Hello').submit()
+
+        //Then
+        expect(Confirmation.getSubtitle()).to.contain('Thank you for your answers, do you wish to submit')
```

Comment:
This is tested in the first test, the 'Then' should be the expectation that we're on the page for editing